### ISSUE TITLE
Temp and Humidity history is not saved

### ISSUE THREAD

**Issue by mrdc** [hannseman/homebridge-mi-hygrothermograph#74]
Hello, 

After installation in May 2019 homebridge-mi-hygrothermograph is not saving any history - "fakeGatoEnabled" is set to true, but "fakegato-history_HygrothermographAccessory.json" has only a few records in it and no new data was added to it since May 2019.
`{"firstEntry":0,"lastEntry":10,"usedMemory":10,"refTime":578787240,"initialTime":1557094440,"history":["noValue",{"time":1557094440,"setRefTime":1},{"temp":25.33,"humidity":46.55,"time":1557094440},{"temp":31.96,"humidity":33.23,"time":1557095040},{"temp":27.34,"humidity":39.71,"time":1557095640},{"temp":25.7,"humidity":42.88,"time":1557098311},{"temp":26.04,"humidity":43.08,"time":1557136028},{"temp":26.03,"humidity":43.11,"time":1557136628},{"temp":24.4,"humidity":41.2,"time":1557138404},{"temp":23.98,"humidity":42.14,"time":1557139004},{"temp":23.53,"humidity":43.18,"time":1557140714}]}`

Other plugins: homebridge-zigbee, mqttthing are saving history without issues.

### Expected Behavior
History should be saved.

### Current Behavior
No history is saved.

### Steps to Reproduce (for bugs)
Unknown.

### Versions
- **homebridge**: 0.4.50
- **homebridge-mi-hygrothermograph**: 2.3.0
- **node**:       v9.11.2
- **OS**:        Linux raspberrypi 4.19.66-v7+ #1253

### Configuration
```json
{
            "accessory": "Hygrotermograph",
            "name": "Temperature & Humidity",
            "timeout": 15,
            "address": "xx:xx:xx:xx:e4:f3",
            "lowBattery": 10,
            "forceDiscovering": true,
            "forceDiscoveringDelay": 120,
            "temperatureName": "Temperature",
            "humidityName": "Humidity",
            "fakeGatoEnabled": true
        },
```

**Comment by mrdc** [comment]
Looks like fakegato part should be updated - I can browse Humidity&Temp history in Eve for iOS 10, but not in the latest Eve app version.

**Comment by hannseman** [comment]
There was a bug in the latest Eve app. A fix was just released (3.8.3). See https://github.com/simont77/fakegato-history/issues/76#issuecomment-532142588.

Update the app and it should start working again.

**Comment by bpelech** [comment]
Hi hannseman. I experience same issue - history is not saving on server. Everything actually works flawlessly except I don't see history data in Elgato Eve nor I can find any json data locally on the server (once I renamed the original fakegato-history_HygrothermographAccessory.json file no other file with this name is created or updated).
